mesonbuild/interpreter.py

```diff
@@ -1402,6 +1492,7 @@ def func_import(self, node, args, kwargs):
     def func_files(self, node, args, kwargs):
         return [mesonlib.File.from_source_file(self.environment.source_dir, self.subdir, fname) for fname in args]
 
+    @permittedKwargs(permitted_kwargs['declare_dependency'])
```

Comment:
Can't we extract this from the name of the function being called instead of duplicating it?